torch/distributed/distributed_c10d.py

```diff
@@ -397,6 +398,7 @@ def init_process_group(backend,
                        world_size=-1,
                        rank=-1,
                        store=None,
+                       pg_options=None,
```

Comment:
We should document this new argument, also, shouldn't all new args go at the end for BC?

Reply:
yeah should be, my fault, updating